aws/resource_aws_route53_recovery_readiness_cell_test.go

```diff
@@ -0,0 +1,238 @@
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckAwsRoute53RecoveryReadinessCellDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAwsRoute53RecoveryReadinessCellConfig(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsRoute53RecoveryReadinessCellExists(resourceName),
+					testAccMatchResourceAttrGlobalARN(resourceName, "cell_arn", "route53-recovery-readiness", regexp.MustCompile(`cell/.+`)),
+					resource.TestCheckResourceAttr(resourceName, "cells.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "parent_readiness_scopes.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "tags.%", "0"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccAwsRoute53RecoveryReadinessCell_nestedCell(t *testing.T) {
+	rNameParent := acctest.RandomWithPrefix("tf-acc-test-parent")
+	rNameChild := acctest.RandomWithPrefix("tf-acc-test-child")
+	resourceNameParent := "aws_route53_recovery_readiness_cell.test_parent"
+	resourceNameChild := "aws_route53_recovery_readiness_cell.test_child"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t); testAccPreCheckAwsRoute53RecoveryReadiness(t) },
+		ErrorCheck:        testAccErrorCheck(t, route53recoveryreadiness.EndpointsID),
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckAwsRoute53RecoveryReadinessCellDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAwsRoute53RecoveryReadinessCellChildConfig(rNameChild),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsRoute53RecoveryReadinessCellExists(resourceNameChild),
+					testAccMatchResourceAttrGlobalARN(resourceNameChild, "cell_arn", "route53-recovery-readiness", regexp.MustCompile(`cell/.+`)),
+				),
+			},
+			{
+				Config: testAccAwsRoute53RecoveryReadinessCellParentConfig(rNameChild, rNameParent),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsRoute53RecoveryReadinessCellExists(resourceNameParent),
+					testAccMatchResourceAttrGlobalARN(resourceNameParent, "cell_arn", "route53-recovery-readiness", regexp.MustCompile(`cell/.+`)),
+					resource.TestCheckResourceAttr(resourceNameParent, "cells.#", "1"),
+					resource.TestCheckResourceAttr(resourceNameParent, "parent_readiness_scopes.#", "0"),
+					testAccCheckAwsRoute53RecoveryReadinessCellExists(resourceNameChild),
+					testAccMatchResourceAttrGlobalARN(resourceNameChild, "cell_arn", "route53-recovery-readiness", regexp.MustCompile(`cell/.+`)),
+					resource.TestCheckResourceAttr(resourceNameChild, "cells.#", "0"),
+				),
+			},
+			{
+				Config: testAccAwsRoute53RecoveryReadinessCellParentConfig(rNameChild, rNameParent),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceNameChild, "parent_readiness_scopes.#", "1"),
```

Comment:
@WyatyPeert is parent_readiness_scopes now resolving > 0? If not, let’s see if the community has any ideas for how to make that attribute refresh.

Reply:
@AdamTylerLynch Yup! As currently written, the scopes resolve to 1 like I'd expect. It just seems a touch misleading to have an additional test step w/ Config solely to get the attribute refreshed.